specification/window.browser.bs

```diff
@@ -0,0 +1,80 @@
+<pre class="metadata">
+Title: window.browser
+Shortname: wecg-browser
+Level: 1
+Group: wecg
+Status: UD
+URL: https://w3c.github.io/webextensions/specification/window.browser.html
+Editor: Patrick Kettner, Google, patrickkettner@google.com
+Abstract: This specification reserves the <code>window.browser</code> namespace for use by WebExtensions.
+Repository: w3c/webextensions
+Complain About: accidental-2119 on, missing-example-ids on
+</pre>
+
+<pre class=link-defaults>
+spec:html; type:attribute; text:window
+</pre>
+
+<pre class="anchors">
+spec: html; urlPrefix: https://html.spec.whatwg.org/multipage/
+    type: dfn; text: valid non-empty URL; url: valid-non-empty-url-potentially-surrounded-by-spaces
+</pre>
+
+<section class="non-normative">
+  ## Introduction {#introduction}
```

Comment:
ah, thanks for pointing that out. That is the syntax for declaring a custom ID in markdown. But bikeshed doens't support it. The correct thing to do would be to define an ID on the section on the line above. But since I added it to be able to link to it later in the doc, and that link was factored out, I will just remove this.